src/Components/Input/input.jsx

```diff
@@ -0,0 +1,8 @@
+export default function Input({label,placeholder,type="text",name}){

```

Comment:
```suggestion
export default function Input({ label, placeholder, type="text", name }) {
```

Reply:
Install the extension:

[ESLint](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint) - Integrates eslint warnings and errors into the editor, which Coursera relies on to keep code consistent, share deprecations, etc.

[Prettier - Code formatter](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) - Auto-formats code

Reply:
I think you can resolve this one.